### PR TITLE
Added util/testing-utils/Tools.js that has a reusable delete schema

### PR DESCRIPTION
**Pull request by ThaiQ**
1. Added util/testing-utils/Tools.js that has a reusable delete schema function for future test cases
2. Replaced all test cases delete schema with util.tools

### CODE REVIEW

File: test/3DPrintingForm.js [hunk 1]
```diff
@@ -69,8 +64,8 @@ describe('3DPrintingForm', () => {
       const form = {
         name: 'pinkUnicorn',
         color: 'Rainbow',
-        contact: 'a@b.c',
-        email: 'a@a.com'
+        contact: 'b@b.c',
```
Comment (evanugarte) [inline test/3DPrintingForm.js]: why do you change these values?

Comment (ThaiQ) [reply test/3DPrintingForm.js]: uniform email throughout the test.

Comment (evanugarte) [reply test/3DPrintingForm.js]: got it thanks

**Comment by evanugarte** [comment]
Resolves #147

**Pull request merged**